docs/user-guide/api-mediation/api-gateway-configuration.md

```diff
@@ -4,74 +4,128 @@
     
-1. Open the file `<Zowe install directory>/components/api-mediation/bin/start.sh`.
-2. Find the line that contains the `-Dapiml.service.allowEncodedSlashes=true` parameter and set the value to `false`.
-3. Restart Zowe&trade. 
+    Use the following procedure reject encoded slashes.
     
-Requests with encoded slashes are now rejected by the API Mediation Layer.
+    **Follow these steps:**
+        
+    1. Open the file `<Zowe install directory>/components/api-mediation/bin/start.sh`.
+    2. Find the line that contains the `-Dapiml.service.allowEncodedSlashes=true` parameter and set the value to `false`.
+    3. Restart Zowe&trade. 
+        
+    Requests with encoded slashes are now rejected by the API Mediation Layer.
        
-### apiml.service.corsEnabled
+* **apiml.service.corsEnabled**
 
-By default, CORS are disabled in the API Gateway for the Gateway routes `api/v1/gateway/**`. Allowing CORS for Gateway is necessary to enable CORS at the service level. Use the following procedure to enable CORS.
-    
-**Follow these steps:**
-     
-1. Open the file `<Zowe install directory>/components/api-mediation/bin/start.sh`.
-2. Find the line that contains the `-Dapiml.service.corsEnabled=false` parameter and set the value to `true`.
-3. Restart Zowe&trade.
-  
-Requests through the Gateway now contain a CORS header. 
+    By default, CORS are disabled in the API Gateway for the Gateway routes `api/v1/gateway/**`. Allowing CORS for Gateway is necessary to enable CORS at the service level. Use the following procedure to enable CORS.
+        
+    **Follow these steps:**
+         
+    1. Open the file `<Zowe install directory>/components/api-mediation/bin/start.sh`.
+    2. Find the line that contains the `-Dapiml.service.corsEnabled=false` parameter and set the value to `true`.
+    3. Restart Zowe&trade.
+      
+    Requests through the Gateway now contain a CORS header. 
 
-### apiml.security.auth.provider
+* **apiml.service.hostname**
 
-By default, the API Gateway uses z/OSMF as an authentication provider. It is possible to switch to SAF as the authentication
-provider instead of z/OSMF. The intended usage of SAF as an authentication provider is for systems without z/OSMF.
-If SAF is used and the z/OSMF is available on the system, the created tokens are not accepted by z/OSMF. Use
-the following procedure to switch to SAF. 
+    This property is used to set the API Gateway hostname.
 
-**Follow these steps:**
-     
-1. Open the file `<Zowe install directory>/components/api-mediation/bin/start.sh`.
-2. Find the line that contains the `-Dapiml.security.auth.zosmfServiceId=zosmf` parameter and replace it with `-Dapiml.security.auth.provider=saf`.
-3. Restart Zowe&trade.
+* **apiml.service.port**
+
+    This property is used to set the API Gateway port.
+
+* **apiml.service.discoveryServiceUrls**
+
+    This property specifies the Discovery Service URL used by the service to register to Eureka.
+
+* **apiml.service.preferIpAddress**
```

Comment:
I would expand a bit on this parameter, behavior and potential risks. We have already solved a few issues with the behavior linked to this. @jandadav maybe you know what it was about?

Reply:
Especially with certificates and SAN names, it can raise problems. We should be explicit about the effects.